app/controllers/admin/articles_controller.rb

```diff
@@ -26,6 +26,15 @@ def create
   def edit
   end
 
+  def update
+    @article.elements.delete_all
+    if @article.update(article_param)
+      render action: :create
```

Comment:
__[rails_best_practices]__
* simplify render in controllers [:ref](http://rails-bestpractices.com/posts/2010/12/12/simplify-render-in-controllers/)